### DISCUSSION TITLE
Hybrid Quantum Neural Networks

### DISCUSSION THREAD

**Discussion by fannafannana** [NVIDIA/cuda-quantum#1633] [General]
qubit_count = 1
When attempting to extend to 2 qubits for a four-class classification task (0, 1, 2, 3), I have encountered numerous issues. Could you please provide updated examples of quantum neural networks (QNNs) that would facilitate development?

**Comment by Gaurang-Belekar** [comment]
Hi @fannafannana , can you please share the structure and what error you are facing?
